### PR TITLE
Code Quality: Improved Widget code

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

1st stage of Widgets cleanup.

Since SideBar and Widget use diffierent data models, added SideBar and Widget prefixes so others could understand what a data model is for easily.

Card means Button.
Container means Expander.
FileTagsContainer means a tag container.

- Renamed: **Widget**DriveCardItem
- Renamed: **Widget**FileTagsCardItem
- Renamed: **Widget**FolderCardItem
- Removed: Some of classes that are referenced from nowhere

**Widget models**
image

## Steps To Validate Changes

1. Launch the app
2. Go to Home page
3. See nothing different from what Files stable shows
4. Right click on any widget card item
5. See the context menu shows correctly

## PR Checklist

- [x] **Development**: #4180
- [x] **Approval**: Have discussed with and got approval from the team[^1]
- [x] **Tests**: Tested accessibility on the local end
- [x] **Deployment**: Deployed the app on the local end
- [ ] **Localization**: Modified en-US string resources[^2]
- [ ] **Co-authors**: _N/A_

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.

### CODE REVIEW

**Comment by 0x5bfa** [comment]
Reduced changed code lines. Mostly renaming and moving classes.

**Comment by 0x5bfa** [comment]
Ready for review

File: src/Files.App/Data/Items/WidgetFileTagCardItem.cs [hunk 1]
```diff
@@ -1,47 +1,64 @@
...
+			=> _associatedStorable is IFolder;
+
 		private IImage? _Icon;
+		public IImage? Icon
+		{
+			get => _Icon;
+			set => SetProperty(ref _Icon, value);
+		}
 
-		[ObservableProperty]
```
Comment (QuaintMako) [inline src/Files.App/Data/Items/WidgetFileTagCardItem.cs:22]: A question to keep up to date with the current codebase: we're getting rid of the `[ObservableProperty]` attribute?

**Review by QuaintMako — COMMENTED** [review]
LGTM, works as expected on my computer. I couldn't manage to find any regression.
A second try might be worth.

File: src/Files.App/Data/Items/WidgetFileTagCardItem.cs (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App/Data/Items/WidgetFileTagCardItem.cs:22]: @yaira2 Are you willing to use those attaributes including `[RelayCommand]`?

**Review by d2dyno1 — APPROVED** [review]
I'd rename the sidebar from "SideBar" to "Sidebar" but it's optional 🙂

**Comment by 0x5bfa** [comment]
Currently, Files has both SideBar and Sidebar, and the VS spell checker warn you if you use Sidebar, so I decided to use SideBar to create a sense of unity with TabBar.

**Comment by yaira2** [comment]
"Sidebar"

**Comment by 0x5bfa** [comment]
TabBar is "Tabbar" or "TabBar"?

**Comment by yaira2** [comment]
"TabStrip"

**Comment by 0x5bfa** [comment]
What😂
Fun fact: TabBar is what you decided to call tho

But whatever it'll be, I'm gonna change all SideBar to be Sidebar.

**Comment by 0x5bfa** [comment]
Ready for review again

**Comment by 0x5bfa** [comment]
@yaira2 I can revert the last commit to be fetched to the last approved state. I wanna rename to Sidebar in the forth coming PR to merge first.

**Comment by 0x5bfa** [comment]
DONE

**Comment by yaira2** [comment]
@0x5bfa can you revert the remaining changes to the `sidebar`?

**Comment by lukeblevins** [comment]
> @yaira2 I can revert the last commit to be fetched to the last approved state. I wanna rename to Sidebar in the forth coming PR to merge first. Plus, It turned out that those classes should be prefixed with Standard\*Item such as StandardDriveItem and StandardLibraryItem because those classes used as standard encapsulated classes everywhere in Files.App (Standard\* respects @lukeblevins's change)

I'd be curious to know which change it was mentioned from me. 😬

**Comment by 0x5bfa** [comment]
> > @yaira2 I can revert the last commit to be fetched to the last approved state. I wanna rename to Sidebar in the forth coming PR to merge first. Plus, It turned out that those classes should be prefixed with Standard*Item such as StandardDriveItem and StandardLibraryItem because those classes used as standard encapsulated classes everywhere in Files.App (Standard* respects @lukeblevins's change)
> 
> I'd be curious to know which change it was mentioned from me. 😬

You renamed ListedItem to StandardItemViewModel in #12601. I found it was pretty valid and I should rename DriveItem, WslDistroItem and so on like this.

**Review by yaira2 — APPROVED** [review]

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**